src/main/java/com/gmail/goosius/siegewar/settings/ConfigNodes.java

```diff
@@ -1073,7 +1073,76 @@ public enum ConfigNodes {
 			"0",
 			"",
 			"# Within the protection radius, players cannot build below this height.",
-			"# The height is relative to the siege banner height");
+			"# The height is relative to the siege banner height"),
+	DOMINATION_AWARDS(
+			"domination_awards",
+			"",
+			"",
+			"",
+			"############################################################",
+			"# +------------------------------------------------------+ #",
+			"# |                  DOMINATION AWARDS                   | #",
+			"# +------------------------------------------------------+ #",
+			"############################################################",
+			""),
+	DOMINATION_AWARDS_ARTIFACT_OFFERS(
+			"domination_awards.artifact_offers",
+			""
+			+ " {potion_of_true_invisibility, 1, 3, potion, invisibility:1:3600:false:false:true} "
+			+ " {sword_of_fiery_knockback, 2, 1, golden_sword, fire_aspect:4, knockback:4} "
+			+ " {helmet of missile shielding, 2, 1, iron_helmet, projectile_protection:10} "
+			+ " {arrow_of_doom, 3, 1, tipped_arrow, instant_damage:20:1:true:true:true} ",
+			"",
+			"# List of Artifact Specifications.",
+			"# There are 2 possible entry types:",
+			"# 1. {weapon/armour, artefact_tier, quantity, material_name, effect_name:power, (optional effect) (optional effect) ...}",
+		    "# 2. {'potion'/'tipped_arrow', artefact_tier, quantity, material_name, effect_name:power:duration:ambient:particles:icon, (optional effect) (optional effect) ...}",
+			"# TODO -arrow",
+			""),

```

Comment:
This section uses artifact while the latter use artefact. There's also an unneeded "" online 1101.

Reply:
Fixed